runtime-ballerina/build.gradle

```diff
@@ -218,6 +219,19 @@ publishing {
     }
 }
 
-ballerinaBuild.dependsOn ":runtime-native:build"
 ballerinaBuild.dependsOn unpackJballerinaTools
```

Comment:
There's a typo in `updateTomlVerions`.

`updateTomlVerions` -> `updateTomlVersions`

```suggestion
updateTomlVerions.dependsOn unpackJballerinaTools
```